src/nflmaps/config.py

```
@@ -0,0 +1,62 @@
+
+__all__ = [
+    "PROJECT_ROOT",
+    "DATA_DIR",
+    "RAW_DIR",
+    "CACHE_DIR",
+    "ASSETS_DIR",
+    "LEGACY_DIR",
+    "DEFAULT_DB_PATH",
+    "USER_AGENT",
+    "database_url",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)